test/scripts/e2e-cp.sh

```diff
@@ -70,6 +70,15 @@ if [ "$OVN_GATEWAY_MODE" == "shared" ]; then
   SKIPPED_TESTS+="Should ensure load balancer service|LGW"
 fi
 
+# FIXME(mk): when in local GW mode, a node is marked as egress and a pod resides on this node, it will fail to talk
```

Comment:
I have a fix for this i was going to work on, where we contact node IPs via geneve using LRPs to redirect to mp0 interfaces on the nodes. I think this solves some other issues as well with talking to other nodes while egress features are configured, and we consider talking to other nodes part of intra cluster traffic and not egress.

Reply:
That will be awesome! I need this for a CU and want it in 4.14.9.